### PR TITLE
New package: himitsu-0.6

### PR DESCRIPTION
**Pull request by Pestdoktor**
requested by  #40466
dependencies tracked at #48783

#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/hare-driver/template [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Template file for 'hare-driver'
+pkgname=hare-driver
+version=0.24.0
+revision=1
+build_style=configure
+make_build_target=".bin/hare docs"
+make_install_target="install-cmd install-mods"
+hostmakedepends="qbe harec scdoc"
+depends="qbe-1.2_1 harec-0.24.0_1"
```
Comment (sug0) [inline srcpkgs/hare-driver/template:9]:
```suggestion
depends="qbe-1.2_1"
```
isn't `harec` only required to bootstrap the compiler?

**Review by sug0 — COMMENTED** [review]
building for any arch other than `aarch64` and `x86_64` should be marked as broken

**Comment by Pestdoktor** [comment]
@sug0 thanks!

**Comment by AnInternetTroll** [comment]
Tested on x86\_64-glibc. I can save, encrypt, and decrypt keys as
expected!

File: srcpkgs/himitsu/patches/01-use-hareflags.patch
Comment (aryalaadi) [inline srcpkgs/himitsu/patches/01-use-hareflags.patch:1]: this could be part of upstream

File: srcpkgs/hare-driver/patches/02-pass-arch-to-harec.patch
Comment (aryalaadi) [inline srcpkgs/hare-driver/patches/02-pass-arch-to-harec.patch:1]: this could also be a part of upstream

**Review by aryalaadi — APPROVED** [review]
Have you submitted the `02-pass-arch-to-harec.patch` and`01-use-hareflags.patch` to upstream? I think you should

**Comment by Pestdoktor** [comment]
Sorry I was away but I'd like to request this be reopened.
Updated my branch to the new upstreams, which also resolves the aforementioned patches.

**Comment by AnInternetTroll** [comment]
Afaik you can't re-open a PR once the branch has been updated, and
someone else has already opened a new PR for hare in #51307

**Pull request closed without merging**